packages/core/evaluate.mjs

```diff
@@ -39,6 +39,10 @@ function safeEval(str, options = {}) {
 
 export const evaluate = async (code, transpiler, transpilerOptions) => {
   let meta = {};
+  //post to iframe parent (like Udels) if it exists...
```

Comment:
could this also be done inside Repl.jsx afterEval ? seems a bit too core to do it here + the handling of the url hash is also there for the normal repl

Reply:
sure I can give that a try